department-of-veterans-affairs/vets-website
# Update Pension income types

## Summary

The list of income types for the question What types of income is updated to match the PDF: Social Security,
Interest or dividend income, Civil Service, Pension or retirement income, and Other income.

- PR to update [json-schema PR](https://github.com/department-of-veterans-affairs/vets-json-schema/pull/833) is merged. 

## Related issue(s)

- https://github.com/department-of-veterans-affairs/va.gov-team/issues/74698

## Testing done

- Run the vets-api and vets-website on branch `pensions-74698-monthly-income`
- Select 'yes' on `/financial/receives-income`
- On `/financial/income-sources`, fill out at least two sources of income completely. 
- Fill out the rest of the form and submit
- The submission should be successful

## Screenshots

|         | Before | After |
| ------- | ------ | ----- |
| Desktop |  <img width="560" alt="Screenshot 2024-01-29 at 9 53 52 AM" src="https://github.com/department-of-veterans-affairs/vets-website/assets/21046714/3eec506a-df8a-4857-9c0e-7b6aaa0e026d"> |     <img width="611" alt="Screenshot 2024-01-29 at 9 55 58 AM" src="https://github.com/department-of-veterans-affairs/vets-website/assets/21046714/e070e77e-87eb-453a-bc93-67e11e575364">  |

## What areas of the site does it impact?

Pension benefits

## Acceptance criteria

### Quality Assurance & Testing


### Error Handling


### Authentication
